test/acceptance/deploy_native_test.go

```diff
@@ -11,13 +11,21 @@ import (
 	"github.com/orbs-network/orbs-network-go/test"
 	"github.com/orbs-network/orbs-network-go/test/acceptance/callcontract"
 	"github.com/orbs-network/orbs-network-go/test/contracts"
+	testKeys "github.com/orbs-network/orbs-network-go/test/crypto/keys"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/stretchr/testify/require"
 	"testing"
 )
 
 func TestDeployNativeContract(t *testing.T) {
-	NewHarness().Start(t, func(t testing.TB, ctx context.Context, network *Network) {
+	NewHarness().
+		WithSetup(func(ctx context.Context, network *Network) {
```

Comment:
consider if this setup should be the default in the harness instead of adding it to each test ?